cli/cmd/create.go:11:2: cannot find package "github.com/vulpemventures/nigiri/cli/builder"

At the build command on Darwin 

`cli/cmd/create.go:11:2: cannot find package "github.com/vulpemventures/nigiri/cli/builder"`
`cli/cmd/create.go:13:2: cannot find package "github.com/vulpemventures/nigiri/cli/helpers"`

--- a/cli/cmd/create.go
+++ b/cli/cmd/create.go
@@ -8,14 +8,8 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/vulpemventures/nigiri/cli/builder"
 	"github.com/vulpemventures/nigiri/cli/config"
-	"github.com/vulpemventures/nigiri/cli/helpers"
 )
-
-var composeBuilder = map[string]func(path string) builder.ComposeBuilder{
-	"regtest": helpers.NewRegtestBuilder,
-}
 
 var CreateCmd = &cobra.Command{
 	Use:    "create",

